# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/.github/ISSUE_TEMPLATE/config.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/.github/ISSUE_TEMPLATE/config.yml b/gfx/wgpu/.github/ISSUE_TEMPLATE/config.yml
deleted file mode 100644
--- a/gfx/wgpu/.github/ISSUE_TEMPLATE/config.yml
+++ /dev/null
@@ -1,8 +0,0 @@
-blank_issues_enabled: false
-contact_links:
-  - name: Issues with shaders
-    url: https://github.com/gfx-rs/naga/issues/new/choose
-    about: Issues with or enhancements for the shader translation.
-  - name: Question about wgpu
-    url: https://github.com/gfx-rs/wgpu-rs/discussions/new
-    about: Any questions about how to use wgpu should go here.